INSTALL

```diff
@@ -73,9 +73,11 @@ Check System Prerequisites
         4xxx  : Use "lspci -d 8086:4941" to check VF devices have been created.
         401xx : Use "lspci -d 8086:4943" to check VF devices have been created.
      * firmware must be available
-        Check that these files exist, if not contact qat-linux@intel.com
+        Check that these files exist:
         /lib/firmware/qat_4xxx.bin or /lib/firmware/qat_4xxx.bin.xz
         /lib/firmware/qat_4xxx_mmp.bin or /lib/firmware/qat_4xxx_mmp.bin.xz
+        If not, clone this repo and copy those files to /lib/firmware:
```

Comment:
The firmware images can be downloaded without cloning the repo with the following commands:
```
wget https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/plain/qat_4xxx.bin
wget https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/plain/qat_4xxx_mmp.bin
```